src/desktop/open_uri.rs

```diff
@@ -171,6 +171,15 @@ impl OpenFileRequest {
         self
     }
 
+    #[must_use]
+    /// Sets the activation token for the application.
+    pub fn activation_token(mut self, activation_token: ActivationToken) -> Self {
+        if activation_token.is_some() {
```

Comment:
that is why I think this should definitely be optional instead of these workarounds